airflow/providers/google/cloud/operators/life_sciences.py

```diff
@@ -22,6 +22,7 @@
 from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
 from airflow.providers.google.cloud.hooks.life_sciences import LifeSciencesHook
+from airflow.providers.google.cloud.links.life_sciences import LifesciencesLink
```

Comment:
What do you think about changing the name to `LifeSciencesLink`, for being in one style with `LifeSciencesHook`?

Reply:
good point